### PR TITLE
Add Copyright and Documentation for Retrieval QA and Vector DB

### PR DESCRIPTION
**Pull request by z7ye**
Adding copyright and merge documentation from feature branch to main.

### CODE REVIEW

**Review by qiuosier — DISMISSED** [review]

**Review by mingkang111 — APPROVED** [review]

**Review by qiuosier — DISMISSED** [review]

File: docs/source/user_guide/large_language_model/retrieval.rst [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+.. code-block:: python3
+
+    from langchain.vectorstores import OpenSearchVectorSearch
+    from ads.llm import GenerativeAIEmbeddings, GenerativeAI
+    import ads
+
+    ads.set_auth("resource_principal")
+
+    oci_embedings = GenerativeAIEmbeddings(
+        compartment_id="ocid1.compartment.oc1..aaaaaaaapvb3hearqum6wjvlcpzm5ptfxqa7xfftpth4h72xx46ygavkqteq",
```
Comment (qiuosier) [inline docs/source/user_guide/large_language_model/retrieval.rst:69]: Remove OCID

File: docs/source/user_guide/large_language_model/retrieval.rst [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        client_kwargs=dict(service_endpoint="https://generativeai.aiservice.us-chicago-1.oci.oraclecloud.com") # this can be omitted after Generative AI service is GA.
+    )
+
+    oci_llm = GenerativeAI(
+        compartment_id="ocid1.compartment.oc1..aaaaaaaapvb3hearqum6wjvlcpzm5ptfxqa7xfftpth4h72xx46ygavkqteq",
```
Comment (qiuosier) [inline docs/source/user_guide/large_language_model/retrieval.rst:74]: Remove OCID

File: docs/source/user_guide/large_language_model/retrieval.rst
Comment (qiuosier) [inline docs/source/user_guide/large_language_model/retrieval.rst:1]: I want to suggest we change the focus of this page from "serialization" to "integration with OCI Generative AI and OpenSearch". We should add more text to explain your retrieval QA example. Maybe we can have sections like:
* OCI Generative Embedding
* Retrieval QA with OpenSearch
* FAISS as Vector DB
* Deployment of Retrieval QA (here we talk about the serialization)

**Review by mingkang111 — APPROVED** [review]

**Review by lu-ohai — APPROVED** [review]

File: docs/source/user_guide/large_language_model/retrieval.rst
Comment (z7ye) [reply docs/source/user_guide/large_language_model/retrieval.rst:1]: will resolve these in a different PR.

File: docs/source/user_guide/large_language_model/retrieval.rst [hunk 3]
```diff
@@ -0,0 +1,194 @@
+.. _vector_store:
+
+########################
+Vector Store integration
+########################
+
+.. versionadded:: 2.9.1
+
+Current version of Langchain does not support serialization of any vector stores. This will be a problem when you want to deploy a langchain application with the vector store being one of the components using data science model deployment service. To solve this problem, we extended our support of vector stores serialization:
```
[the remark this answers is not in the document]
Comment (z7ye) [reply docs/source/user_guide/large_language_model/retrieval.rst:9]: sure. will resolve this in a different PR.

File: docs/source/user_guide/large_language_model/retrieval.rst (hunk 1 above)
Comment (z7ye) [reply docs/source/user_guide/large_language_model/retrieval.rst:69]: will do.

File: docs/source/user_guide/large_language_model/retrieval.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (z7ye) [reply docs/source/user_guide/large_language_model/retrieval.rst:9]: done

File: docs/source/user_guide/large_language_model/retrieval.rst (hunk 2 above)
Comment (z7ye) [reply docs/source/user_guide/large_language_model/retrieval.rst:74]: done

**Pull request merged**